rst/data.rst

```diff
@@ -25,6 +25,35 @@
    (The :samp:`fsname=<foo>` option is guaranteed never to be included in the
    default options, so you can always safely add it to the set).
 
+   The default options are:
+
+   * ``default_permissions`` enables permission checking by kernel.
+     Without this any umask (or uid/gid) would not have an effect.
```

Comment:
isn't this a pitfall just waiting for somebody?

and why does one need to explicitly give it, if it is (or should be) "default"?